chandra_aca/transform.py

```diff
@@ -188,10 +226,57 @@ def _poly_convert(y, z, coeffs, t_aca=None):
 
     newy = np.sum(coeffs[:, 0] * poly.transpose(), axis=-1)
     newz = np.sum(coeffs[:, 1] * poly.transpose(), axis=-1)
+    if shape:
+        newy.shape = shape
+        newz.shape = shape
 
     return newy, newz
 
 
+def radec_to_yagzag(ra, dec, q_att):
+    """
+    Given RA, Dec, and pointing quaternion, determine ACA Y-ang, Z-ang.  The
+    input ``ra`` and ``dec`` values can be 1-d arrays in which case the output
+    ``yag`` and ``zag`` will be corresponding arrays of the same length.
+
+    This is a wrapper around Ska.quatutil.radec2yagzag but uses arcsec instead
+    of deg for yag, zag.
+
+    :param ra: Right Ascension (degrees)
+    :param dec: Declination (degrees)
+    :param q_att: ACA pointing quaternion
+
+    :returns:  yag, zag (arcsec)
+    """
+    from Ska.quatutil import radec2yagzag
```

Comment:
Should we just put the Ska.quatutil functions in here and put wrappers in Ska.quatutil?

Reply:
The Ska.quatutil funcs use the eci transform functions, so it's not that easy.  Anyway, this does the job with just one package update (which comes for free anyway), vs. two package updates the other way.